install/helm/agones/templates/serviceaccounts/sdk.yaml

```diff
@@ -23,6 +23,9 @@ metadata:
     chart: {{ template "agones.chart" $ }}
     release: {{ $.Release.Name }}
     heritage: {{ $.Release.Service }}
+{{- if .Values.agones.serviceaccount.sdk.annotations }}
```

Comment:
This if needs a matching `{{- end }}`.

Reply:
From the build error: `Error: parse error at (agones/templates/serviceaccounts/sdk.yaml:73): unexpected EOF`

Reply:
Thanks for finding that.